README.md

```diff
@@ -51,3 +50,7 @@ You may also run the tests. For details on testing see the
 documentation.
 
 > npm test
+
+## Next Steps
+
+Developer setup: https://github.com/mellychat/webapp/blob/mainline/docs/developersetup.md
```

Comment:
Let's link over to it instead:

`1. [Setup your dev environment](https://github.com/mellychat/webapp/blob/mainline/docs/developersetup.md)`